### PR TITLE
More single alias

### PR DESCRIPTION
**Pull request by Ramit110**

### CODE REVIEW

**Comment by Ramit110** [comment]
Honestly, point out as much as possible for this one, it's pretty iffy.

**Comment by tboby** [comment]
I think it might be easier to keep the single_alias in the same file when it's **this** specific.

File: aliases.cwt [hunk 1]
```diff
@@ -194,4 +194,12 @@ single_alias[any_plotter_trigger_clause] = {
...
 
     alias_name[trigger] = alias_match_left[trigger]
+}
+
+single_alias[law_ai_chance] = {
+    ### Base chance. 1.0 is 100%.
+    factor = float
+
+    ## replace_scope = { this = title root = title }
+    alias_name[multiplicative_modifier] = alias_match_left[multiplicative_modifier]
```
Comment (tboby) [inline aliases.cwt:204]: Are this and `decision_ai_will_do_clause` the same? (Other than the scopes, which could be set in the law block)

Comment (Ramit110) [reply aliases.cwt:204]: I'm not sure if the ``## replace_scope = { this = title root = title }`` is important or not so I made it a new alias.

**Pull request merged**